app/models/item.rb

```diff
@@ -0,0 +1,2 @@
+class Item < ApplicationRecord
```

Comment:
Same issue as in https://github.com/goozzik/AppleMarket/pull/4#discussion_r301420435